manifests/alertmanager-alertmanager.yaml

```diff
@@ -3,12 +3,22 @@ kind: Alertmanager
 metadata:
   labels:
     alertmanager: main
+    app.kubernetes.io/component: router
+    app.kubernetes.io/name: alertmanagermain
+    app.kubernetes.io/part-of: kube-prometheus
+    app.kubernetes.io/version: v0.21.0
   name: main
   namespace: monitoring
 spec:
   image: quay.io/prometheus/alertmanager:v0.21.0
   nodeSelector:
     kubernetes.io/os: linux
+  podMetadata:
+    labels:
+      app.kubernetes.io/component: router
+      app.kubernetes.io/name: alertmanagermain
```

Comment:
Why alertmanagermain and not alertmanager?

Reply:
typo, it should be `alertmanager-main` as this is the StatefulSet name.